examples/README.md

```diff
@@ -0,0 +1,26 @@
+# SigOpt API Examples
+For the complete API documentation, visit [https://sigopt.com/docs](https://sigopt.com/docs).
+## Python
+Check out our `basic` python example and implement your own function to evaluate. Dive deeper with our `parallel` example, which runs multiple optimization loops at the same time.
```

Comment:
This is the screen view they'll see when they're viewing the readme, so I hope it'll be clear which examples we're referring to. If this gets more complicated we can add links but I think people who reach this point will feel comfortable diving in.
![screen shot 2016-02-16 at 11 50 33 am](https://cloud.githubusercontent.com/assets/913838/13089000/8cd202d0-d4a3-11e5-8ec2-6558273016d6.png)